src/OpenLoco/Config.cpp

```diff
@@ -30,17 +32,70 @@ namespace OpenLoco::Config
         return _new_config;
     }
 
+    constexpr uint8_t _defaultMaxVehicleSounds[3] = { 4, 8, 16 };
+    constexpr uint8_t _defaultMaxSoundInstances[3] = { 6, 8, 10 };
+    constexpr uint8_t _defaultPreferredCurrency[16] = { 0x82, 0, 0, 0, 0x43, 0x55, 0x52, 0x52, 0x44, 0x4F, 0x4C, 0x4C, 0, 0, 0, 0 };
```

Comment:
This reads CURRDOLL. Would it be possible to use e.g.:
```suggestion
    constexpr uint8_t _defaultPreferredCurrency[16] = { 0x82, 0, 0, 0, 'C', 'U', 'R', 'R', 'D', 'O', 'L', 'L', 0, 0, 0, 0 };
```

Reply:
What is currdoll?

Reply:
CURRency DOLLar?

Reply:
Ahh that makes a lot of sense.